tests/cypress/integration/stateWorkflow/stateSubmissionForm/documents.spec.ts

```diff
@@ -18,7 +18,9 @@ describe('documents', () => {
                 'documents/trussel-guide.pdf'
             )
             cy.findByText(/0 complete, 1 error, 1 pending/).should('exist')
-            cy.waitForDocumentsToLoad()
```

Comment:
why did you drop .waitForDocumentsToLoad() in favor of a wait()?

Reply:
The table, unlike the list, doesn't show an image next to the completed upload.  So, while loading, there's a spinner, and then, when loading is finished, nothing.  waitForDocumentsToLoad relies on the list behavior: check the image, see if it's loading.  In the table, you can't check the class on the image, because the image might not be there.  And you can't do a simple check that the image is _not_ there, because that will always be true before the page has fully rendered.  So I wait for the page to render, then make sure the spinner isn't still there.